fastccd_support_ioc/delay_generator_ioc.py

```diff
@@ -100,16 +100,16 @@ async def state(self, instance, value):
 
         return value
 
-    initialize = pvproperty(value=0, dtype=int, put=initialize)
-    shutdown = pvproperty(value=0, dtype=int, put=shutdown)
+    initialize = pvproperty(value=0, dtype=int, put=_initialize)
+    shutdown = pvproperty(value=0, dtype=int, put=_shutdown)
 
     @state.startup
     async def state(self, instance, async_lib):
-        await self.initialize(None, None)
+        await self._initialize(None, None)
 
     @state.shutdown
     async def state(self, instance, async_lib):
-        await self.initialize(None, None)
+        await self._shutdown(None, None)
```

Comment:
CL is just a clear instrument, so not really a shutdown, should we rename it _clear instead?

Reply:
Renamed it to "reset". Does running initialize at startup and reset at [IOC] shutdown make sense?

Reply:
I don't think so. Roland should answer that. I only used it when I god bad command errors. But when the device is shutdown it should not matter.